src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_help.py

```diff
@@ -287,6 +287,22 @@
         - name: Update elastic pool with zone redundancy explicitly disabled
           text: az sql elastic-pool update -g mygroup -s myserver -n mypool -z false
     """
+helps['sql failover-group'] = """
+    type: group
+    short-summary: Manage SQL Failover Groups.
+"""
+helps['sql failover-group create'] = """
+    type: command
+    short-summary: Creates a failover group.
+"""
+helps['sql failover-group update'] = """
+    type: command
+    short-summary: Updates the parameters of the failover group.
+"""
+helps['sql failover-group set-primary'] = """
+    type: command
+    short-summary: Executes a failover of a failover group.
```

Comment:
Can we make this more clear and similar to set-primary parameter in GeoDr links?
`Set the primary replica database by failing over from the current primary replica database.`